src/api/main/meta.ts

```diff
@@ -122,9 +121,20 @@ export type CustomTokenRequest = {
   address: string
 }
 
+/**
+ * The request to search for custom tokens.
+ * @property {string} query - The search query string.
+ */
+export type SearchCustomTokensRequest = { query: string }
+
+/**
+ * The response for a custom token search.
+ */
+export type SearchCustomTokensResponse = Token[]
```

Comment:
We had an informal agreement with the API team to structure the response this way. Could you confirm with them whether we should keep this endpoint as it is or modify it?

```suggestion
export type SearchCustomTokensResponse = {
  result: Token[]
}
```